LICENSE

```diff
@@ -0,0 +1,28 @@
+BSD 3-Clause License
+
+Copyright (c) 2023, COSMO
```

Comment:
```suggestion
Copyright (c) 2023, equistore developers
```

and we could add an AUTHORS file if we want (or just use git to track authorship)

Reply:
Yes an AUTHOR file is a good idea.